Data_Structures_Questions.md

```diff
@@ -3,23 +3,24 @@ For each of the methods associated with each data structure, classify it based o
 ## Linked List
 
 1. What is the runtime complexity of `add_to_tail`?
-  
+    O(c)
     a. What if our list implementation didn't have a reference to the tail of the list in its constructor? What would be the runtime of the `add_to_tail` method?
+    O(n)
 
 2. What is the runtime complexity of `remove_head`?
-
+    O(c)
 3. What is the runtime complexity of `contains`?
-
+    O(n)
 4. What is the runtime complexity of `get_max`?
-
+    O(n)
 ## Queue
 
 1. What is the runtime complexity of `enqueue`?
-
+    O(c)
 2. What is the runtime complexity of `dequeue`?
-
+    O(c)
 3. What is the runtime complexity of `len`?
-
+    O(n)
```

Comment:
Ordinarily, finding a length would be `O(n)`. However, we have a `size` property in the queue, which is updated with the current size of the queue every time an element is added or removed. We, then, only need to access this attribute to get the `len` of the queue. Does this give you a different perspective on the Big O notation of `len`?